lighthouse-cli/bin.js

```diff
@@ -132,9 +142,9 @@ async function begin() {
     });
   }
 
-  return runLighthouse(url, cliFlags, configJson);
+  return runLighthouse(urlUnderTest, cliFlags, configJson);
 }
 
-module.exports = {
-  begin,
+export {
+  begin
```

Comment:
not an emergency, but it feels like we probably want
```js
'comma-dangle': [2, {
  imports: 'always-multiline',
  exports: 'always-multiline',
}]
```
seems weird not to have one

Reply:
agreed, will put up a separate PR